Flutter_Weather/jsons/mockjsondata.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This is a json data model file,  not a mock data file. So the name should be 'weather.json'